style.css

```diff
@@ -1,595 +1,1312 @@
+    padding: 4px 12px;
+    width: 45px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  #javascript {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 12px;
+    width: 89px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    flex: none;
+    order: 2;
+  }
+
+  .Click {
+    width: 295px;
+    height: 72px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    flex: none;
+    order: 3;
+    flex-grow: 0;
+  }
+
+  .button-1 {
+    width: 124px;
+    height: 48px;
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    background-color: white;
+    color: #396df2;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    cursor: pointer;
+    transition: all 3s;
+  }
+
+  .About-myself {
+    width: 100%;
+    top: 4535px;
+    background: #fff;
+    height: 924px;
+    border-top-right-radius: 50px;
+  }
+
+  .About {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    margin: 0 0 -329px 24px;
+    padding: 82px 0 0 145px;
+    gap: 24px;
+    width: 447px;
+    height: 392px;
+    top: 3753px;
+  }
+
+  .About h2 {
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    display: flex;
+    align-items: center;
+    color: #091e42;
+  }
+
+  .About p {
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .About h3 {
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    flex: none;
+    flex-grow: 0;
+  }
+
+  .Skills {
+    padding: 0;
+    width: 684px;
+    top: 3253px;
+    float: right;
+  }
+
+  #GE {
+    margin-left: -24px;
+  }
+
+  .button-2 {
+    width: 124px;
+    height: 48px;
+    font-family: "Poppins", sans-serif, "Times New Roman", Times, serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    background-color: white;
+    color: #396df2;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    margin-left: 24px;
+    cursor: pointer;
+    transition: all 3s;
+  }
+
+  .Languages {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px;
+    gap: 12px;
+    justify-content: center;
+    width: 414px;
+    height: 144px;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin: 40px 0 0 -12px;
+  }
```

Comment:
- [ ] Kindly take note of the styling of the list of technologies as the previous reviewer pointed out. You need to pad it a bit.

## Your design 

![b2](https://user-images.githubusercontent.com/103386183/199930283-13e7d07b-8945-4ccf-b701-3ded1c5038a6.png)

## Figma desgin

![b3](https://user-images.githubusercontent.com/103386183/199930641-29e11e19-ed20-4e7d-bdf3-ae723830eb68.png)